test/data/BinarySolutionTabulatedThermo.yaml

```diff
@@ -43,6 +43,16 @@ phases:
       19.2971, 19.2977, 16.2213, 19.298, 19.2978, 19.2945, 19.2899, 19.2877,
       19.2882, 19.2882, 19.2882, 19.2882, 19.2882, 19.2885, 19.2876, 19.2837,
       19.2769, 19.285, 19.31, 19.3514]
+    molarVolume: [34.582988, 34.677730, 34.776275, 34.875760, 34.973980, 35.069268,
```

Comment:
In the YAML format, we generally use lowercase words separated by spaces, so this should be `molar-volume`.